internal/brokers/examplebroker/broker.go

```diff
@@ -365,26 +376,29 @@ func getSupportedModes(sessionInfo sessionInfo, supportedUILayouts []map[string]
 func getMfaModes(info sessionInfo, supportedModes map[string]map[string]string) map[string]map[string]string {
 	mfaModes := make(map[string]map[string]string)
 
-	if authMode, exists := supportedModes["phoneack1"]; exists && info.selectedMode != "phoneack1" {
-		mfaModes["phoneack1"] = authMode
-	}
-	if authMode, exists := supportedModes["totp_with_button"]; exists && info.selectedMode != "totp_with_button" {
-		mfaModes["totp_with_button"] = authMode
-	}
-	if authMode, exists := supportedModes["fidodevice1"]; exists && info.selectedMode != "fidodevice1" {
-		mfaModes["fidodevice1"] = authMode
+	supportedMfaModes := []string{"phoneack1", "totp_with_button", "fidodevice1"}
+	for _, mode := range supportedMfaModes {
```

Comment:
You can directly range over the []string, no need to name it as we don’t reuse it afterwards